src/test/resources/Facebook/FacebookLogin.feature

```diff
@@ -0,0 +1,31 @@
+Feature: FacebookLogin
+
+  Scenario: Verify user gets error message 'find your account and log in ' when user enters invalid credentials in facebook page
+    Given  I launch facebook website
+    When I enter '%#&&*&&' as login email
+    And I enter 'abcd@1234' as login password
+    And I click on login button in facebook page
+    Then I verify error message 'find your account and log in' is displayed
+
+
+  Scenario: Verify user gets error message 'Please choose a gender' when user Sign up facebook without entering gender details
```

Comment:
At each scenario we can use specific tag like @loginOne so that we can run any specific tag if required.